### ISSUE TITLE
Careful not to use double (a floating point type) for currency.

### ISSUE THREAD

**Issue by Hobart** [kittygomeowr/JetsProject#1]
https://github.com/kittygomeowr/JetsProject/blob/ab62ac969d5e9570fd94b48d39041483f05705cc/src/Jets/SpaceCraft.java#L7

Because everything is terrible, floating point values (which are binary) can't exactly store base 10 fractions like 0.10 , and currency math will look like it's working but be subtly going crazy and hard to debug later.

Example of awfulness: https://gist.github.com/Hobart/cd675d992064d5f3bd8e9ed2ee71ed4d

More on StackOverflow: https://stackoverflow.com/q/3730019

**Comment by RosanneUssery** [comment]
Yes! I had encountered this before, but had done wibbly math to make it work better later. I think for this project (since it's for a menu in the future) money can be an int because no one has pockets for their space pennies. 

That example will come in useful later though!

Ty!
